shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -25,17 +26,23 @@
  */
 public class ShortCircuitParameters extends AbstractExtendable<ShortCircuitParameters> {
```

Comment:
Note that we should add some javadoc for all parameters: but can wait for API to be validated ?